draft-regext-rdap-extensions.md

```diff
@@ -216,16 +308,16 @@ The example given in [@!RFC9083] is as follows:
     }
 
 In this example, the extension identified by "lunarNIC" is prepended
-to the names of both a JSON string and a JSON array.
+to the member names of both a JSON string and a JSON array.
 
-As [@!RFC9083, section 4.1] requires the use of the `rdapConformance` data structure
-and the `objectClassName` string is required of all object class instances,
-the complete example from above would be:
+As [@!RFC9083, section 4.1] requires the use of the "rdapConformance"
+data structure, and the "objectClassName" string is required of all
+object class instances, the complete example from above would be:
 
     {
       "rdapConformance": [
         "rdap_level_0",
-        "lunarNIC"  
+        "lunarNIC"
```

Comment:
In this example, we seemingly extend the "domain" object class definition by including JSON members for the new "lunarNIC" extension. IIRC, there is no place in the current RDAP spec -- RFC 7480 and RFC 9083 in particular -- that this inheritance model is explicitly mentioned. Would it help to clarify this?

Reply:
> What does "inheritance model" mean?  Is it like inheritance in a
Java-like language, so "Domain" would be the superclass and
"DomainWithLunarNICAttributes" would be the subclass?

Yah but it might be me overthinking as a developer. :) Since JSON is not as strongly typed as Java, the JSON data for an extension would be simply juxtaposition-ed next to an object class' data in an RDAP response. Ignore this. :)

Reply:
Right, agreed.